### ISSUE TITLE
Inserting multiple endpoints when running azure create vm ?

### ISSUE THREAD

**Issue by Kahelek** [Azure/azure-xplat-cli#1922]
when running a azure vm create with options, I run into a problem which is easily solved, but creates more problems .. 

error:   Deployment traffictrace uses ReservedIP RIP Cloud but does not contain any endpoints. Deployment must contain at least one endpoint in order to use a ReservedIP.
info:    Error information has been recorded to C:\Users\kkoenig\.azure\azure.err
error:   vm create command failed

The solution, formerly posted at: https://github.com/Azure/azure-xplat-cli/issues/1461  ->> was to add the "-e" option, which inserts a singular endpoint on port 22.

While this solves the problem of immediately creating the VM, I cannot go back after the fact and modify the endpoint.

C:\Users\kkoenig\AppData\Roaming\npm\node_modules\azure-xplat-cli>azure vm endpoint create-multiple -c "80:80:TCP::::::::","443:443:TCP::::::::","22:22:TCP::::::::" Fortifire
info:    Executing command vm endpoint create-multiple

+ Getting virtual machines
+ Reading network configuration
error:   this VM already has a tcp load balancer port 22. public port and protocol together should be unique
info:    Error information has been recorded to C:\Users\kkoenig\.azure\azure.err
error:   vm endpoint create-multiple command failed

Basically, I need a way to pipe (or insert) all of the extra "multiple" endpoint creation logic into the vm create statement at the original time of running the VM create statement. I do not see a way to do this?

Thanks
